netket/graph/custom_graph.py

```diff
@@ -0,0 +1,99 @@
+from abstract_graph import PyAbstractGraph
+
+import numpy as _np
+import networkx as _nx
+
+class PyCustomGraph(PyAbstractGraph):
```

Comment:
I would call this just ```Graph``` (or ```PyGraph```, for the time being)

Reply:
I will call it `Graph`